packages/deepsignal/test/index.test.tsx

```diff
@@ -1,11 +1,9 @@
-import "@preact/signals";
+// @ts-ignore
 import { createElement, createRef, render } from "preact";
 import { setupRerender } from "preact/test-utils";
 import { deepSignal, useDeepSignal, type DeepSignal } from "deepsignal";
 
-const sleep = (ms?: number) => new Promise(r => setTimeout(r, ms));
```

Comment:
Just wondering why this is not needed anymore. 🤔

Reply:
My tests where pretty much a copy/paste from the original preact signal ones, so that came with the paste. I deleted it because the tests pass without it 🙂🤷‍♂️